src/components/Dashboard/ProgressBar/ProgressBar.scss

```diff
@@ -1,9 +1,80 @@
-@import '../../../scss/_variables.scss';
-@import '../../../scss/_mixins.scss';
+@import "../../../scss/_variables.scss";
+@import "../../../scss/_mixins.scss";
 
 .ProgressBar {
   @include containerStyle();
-  @include flex(space-between);
-  height: 60px;
+  @include flex(center);
   margin-bottom: 20px;
-}
\ No newline at end of file
+  height: 100px;
+
+  .Progressbar__list {
+    align-items: center;
+    @include flex(center);
+    width: 100%;
+
+
+    li {
+      color: #dcd8d8;
+      list-style-type: none;
+      text-align: center;
+      text-transform: uppercase;
+      position: relative;
+      width: 100%;
+    }
+
+    li:before {
```

Comment:
We could take better use of sass nesting here. Instead of repeating li and so on we can nest it, the same as you nested `ProgressBar` and `ProgressBar__list`. Just with `&` selector